Docs: document contact points for unified alerting (#34983)
grafana/grafana: docs/sources/alerting/unified-alerting/contact-points.md

1. Find the contact point you want to edit in the contact points table and click the **trash can icon** on the right side.
1. A confirmation dialog will open. Click **Yes, delete**.

**Note** You will not be able to delete contact points that are currently used by any notification policy. If you want to delete such contact point, you will have to first go to [notification policies]({{< relref "./notification-policies.md" >}}) and delete the policy or update it to use another contact point.

## List of notifiers supported by Grafana

Name | Type 
-----|-----
[DingDing](#dingdingdingtalk) | `dingding`
[Discord](#discord) | `discord`
[Email](#email) | `email`
[Google Hangouts Chat](#google-hangouts-chat) | `googlechat`
[Kafka](#kafka) | `kafka`
Line | `line`
Microsoft Teams | `teams`
[Opsgenie](#opsgenie) | `opsgenie`
[Pagerduty](#pagerduty) | `pagerduty`
Prometheus Alertmanager | `prometheus-alertmanager`
[Pushover](#pushover) | `pushover`
Sensu | `sensu`
[Sensu Go](#sensu-go) | `sensugo`
[Slack](#slack) | `slack`
Telegram | `telegram`
Threema | `threema`
VictorOps | `victorops`
[Webhook](#webhook) | `webhook`
[Zenduty](#zenduty) | `webhook` 

> ## Managing contact points for an external alertmanager

Reviewer: ```suggestion
## Managing contact points for an external Alertmanager
```

Grafana alerting UI supports managing external alertmanager configuration. Add an [alertmanager datasource]({{< relref "../../datasources/alertmanager.md" >}}) and a dropdown will appear at the top of the page, allowing you to select either `Grafana` or an external alertmanager. 

> {{< figure max-width="40%" src="/static/img/docs/alerting/unified/contact-points-select-am-8-0.gif" caption="Select alertmanager" >}}

Author: git checkout website repo and run `make docs-local-static`. see https://github.com/grafana/grafana/blob/main/docs/README.md

> ### Editing alertmanager global config

Reviewer: ```suggestion
### Editing global configuration
```

> To edit global configuration options for an alertmanager, like SMTP server that is used by default for all email contact types:

Reviewer: ```suggestion
Some options can be global for all contact point types. An example of these is the SMTP server that is used by default for all email contact types. These options can be modified by:
```

1. In the Grafana side bar, hover your cursor over the **Alerting** (bell) icon and then click **Contact points**.

> 1. In the dropdown at the top of the page, select a alertmanager datasource.

Reviewer: ```suggestion
1. In the dropdown at the top of the page, select an Alertmanager datasource.
```

1. Click **Edit global config** button at the bottom of the page.
1. Fill out the form and click **Save global config**.

> **Note** this is only for external alertmanagers. Some global options for Grafana contact types, like email settings, can be configured via [Grafana configuration]({{< relref "../../administration/configuration.md" >}}).

Reviewer: ```suggestion
**Note** this is only for external Alertmanagers. Some global options for Grafana contact types, like email settings, can be configured via [Grafana configuration]({{< relref "../../administration/configuration.md" >}}).
```
